src/components/Founder.tsx

```diff
@@ -57,10 +52,26 @@ export default function Founder() {
     }
   }, [])
 
-  return (
-    <div className={container}>
-      <KetlLogo />
-      <Description />
-    </div>
-  )
+  useEffect(() => {
+    if (!data) return
+    void createFounderProof(data)
+    .then((proof) => {
+      postWebViewMessage({
+        data: proof,
+        type: Messages.GetFounderProof,
+      })
+    })
+    .catch((e) => {
+      postWebViewMessage({
+        data: {
+          message: `Can't generate valid proof with this token`,
+          e: JSON.stringify(e, Object.getOwnPropertyNames(e)),
+        },
+        type: Messages.GetFounderProofError,
+      })
+    })
```

Comment:
It's look strange, specially with `void`. Can we move it to `helpers` or even better, use `useCallback` here?